src/Orleans/Orleans.csproj

```diff
@@ -353,15 +356,13 @@
     <None Include="..\Orleans.snk">
       <Link>Orleans.snk</Link>
     </None>
-    <None Include="packages.config"/>
+    <None Include="packages.config" />
   </ItemGroup>
   <Import Project="$(MSBuildToolsPath)\Microsoft.CSharp.targets" />
   <PropertyGroup>
     <CodeGenToolReferences>System.Data.dll;$(SolutionDir)packages\Microsoft.Data.Services.Client.5.6.0\lib\net40\Microsoft.Data.Services.Client.dll;</CodeGenToolReferences>
   </PropertyGroup>
-  <Target Name="GenerateVersionNumber" BeforeTargets="VersionInit" 
-          Condition="'$(BuildingInsideVisualStudio)' != 'true'" 
-          Outputs="$(SolutionDir)Build\Version.txt">
+  <Target Name="GenerateVersionNumber" BeforeTargets="VersionInit" Condition="'$(BuildingInsideVisualStudio)' != 'true'" Outputs="$(SolutionDir)Build\Version.txt">
```

Comment:
BTW, if you are using SourceTree app it has really nice feature that lets you selectively "revert" parts of a change in a file before you do the local commit. 
So for example you could remove this part of the change in this file but leave the other change below intact. 
See the lower right panel of the SourceTree commit form.